storage/innobase/xtrabackup/src/backup_mysql.h

```diff
@@ -272,6 +272,9 @@ void dump_innodb_buffer_pool(MYSQL *connection);
 
 void check_dump_innodb_buffer_pool(MYSQL *connection);
 
+using dd_tablespaces = std::unordered_map<std::string, space_id_t>;
+dd_tablespaces build_tablespace_space_id_map(MYSQL *conneciton);
```

Comment:
move them to dict0dd.h

Reply:
why do we want to move in dict0dd.h?

Reply:
All dd maps, dd searches, SDI loading are related to dictionary. Hence dict0dd.h

Reply:
To query database we use  function in  backup_mysql.cc. That is the reason I added there.
To use `xb_mysql_query`. I would to include few more header in dict0dd.h ?

Reply:
sorry, I meant dict0dd.cc. I think the query part can remain here. you can send space_id from this file to a function in dict0dd.cc

Reply:
we can not include `backup_mysql.h` inside dict0dd.cc. Compilation throws error. As part of query we are building the set/map
 Maybe we need to add cmake flag 🤔

Reply:
it is not required to move query to dict0dd.cc. Keep the query in backup_mysql.cc and send the space_id to a function in dict0dd.cc dict_store_map(space_id).